client/app/views/menu.coffee

```diff
@@ -12,6 +15,23 @@ module.exports = class MenuView extends ViewCollection
 
     events: ->
         'click .calendars': 'toggleDropdown'
+        'click .calendar-add': 'onAddCalendar'
+
+    onAddCalendar: ->
+        @tag = app.tags.getOrCreateByName "new calendar"
+        # Since an event is needed to create a calendar, let's make a false one
+        calendarEvent = new Event
```

Comment:
Before creating a new event, you could check that an even doesn't already have the tag (even if it's an edge-case).